doc/tools/docgram/orderedGrammar

```diff
@@ -1802,12 +1802,16 @@ ltac2_conversion: [
 | term "with" term      (* ltac2 plugin *)
 ]
 
+q_orient: [
+| OPT [ "->" | "<-" ]
+]
+
```

Comment:
Could you move this below `ltac2_oriented_rewriter`?  You'll need to adjust the `insertprodn` in the rst file.
When possible, we put referenced nonterminals below the nonterminals that reference them.

Reply:
Done